src/index.ts

```diff
@@ -32,16 +33,24 @@ interface UserInput {
   birthDate: string;
 }
 
+const prisma = new PrismaClient();
+
 const resolvers = {
   Query: {
     hello: () => 'Hello World',
   },
   Mutation: {
-    createUser: (_, args: { data: UserInput }) => {
-      return {
-        id: 1,
-        ...args.data,
-      };
+    createUser: async (_, args: { data: UserInput }) => {
+      const userData = args.data;
```

Comment:
aqui, ficou legal vc separar para nao ter que passar sempre args.data.name, args.data.email, etc

Daí tem duas coisas que vc pode fazer, vou deixar para curiosidade =)
1. aqui é uma coisa legal pra mudar na linha 47+, a de numero 2 vou deixar na linha 47
```suggestion
      const { name, email, password, birthDate } = args.data;
```

Reply:
Perfeito Daniel, acabei fazendo pelo 1 porque acho que facilita no tratamento dos inputs posteriormente, obrigado